webservices/resources/reports.py

```diff
@@ -531,5 +532,6 @@ def get_reports(self, committee_id, reports_class, args, page_num, per_page):
             reports = reports.filter(reports_class.cycle.in_(args['cycle'].split(',')))
 
         count = reports.count()
+        import ipdb; ipdb.set_trace()
```

Comment:
Was reconciling branches and forgot to take debugger out here. This is fixed, and I just merged #642 here to deal with merge conflicts.